resources/js/components/passport/AuthorizedClients.vue

```diff
@@ -0,0 +1,107 @@
+<style scoped>
```

Comment:
This file should be removed IAW https://github.com/beauraines/dog-walker/issues/73#issuecomment-627025983